baseapp-profiles/baseapp_profiles/graphql/mutations.py

```diff
@@ -50,9 +50,18 @@ def create(self, validated_data):
 
 
 class ProfileUpdateSerializer(BaseProfileSerializer):
+    phone_number = serializers.CharField(required=False)
+
+    class Meta(BaseProfileSerializer.Meta):
+        fields = BaseProfileSerializer.Meta.fields + ("phone_number",)
+
     def update(self, instance, validated_data):
         url_path = validated_data.pop("url_path", None)
+        phone_number = validated_data.pop("phone_number", None)
         instance = super().update(instance, validated_data)
+        if phone_number and hasattr(instance.owner, "phone_number"):
```

Comment:
will it be possible to have `instance.owner` without a `phone_number` attribute?

Reply:
For the moment, yes